packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/index.js

```diff
@@ -27,6 +27,10 @@ const recordOptions = {
   height: `${recordOptionsHeight}px`,
   key: 'recordOptions',
   renderRow: props => <ResultOptionsBar {...props} />,
+  style: {
+    paddingLeft: '1em',
+    paddingRight: '1em',
```

Comment:
Ida, this should be fixed too according to this: https://github.com/DINA-Web/dina-collections/pull/320#discussion_r236574238

I don't want to make the same comment several times, I'm expecting you to apply the fix in all places it's needed :-)

Reply:
Yes, but does that Lisa want it be in the parent div and you comment that: "In that case maybe it has to be here. "